app/admin.go

```diff
@@ -62,20 +58,48 @@ func (a *App) GetLogsSkipSend(page, perPage int) ([]string, *model.AppError) {
 
 		defer file.Close()
 
-		offsetCount := 0
-		limitCount := 0
-		scanner := bufio.NewScanner(file)
-		for scanner.Scan() {
-			if limitCount >= perPage {
-				break
+		var newLine = []byte{'\n'}
+		var lineCount int
+		var searchPos = int64(-1)
```

Comment:
If you're gonna keep `searchPos`, you could make it a constant (`const searchPos = -1`). Otherwise, my static analysis tools will complain that this "variable" isn't actually variable. And at a glance it looks like a mistake since it's never updated.